### ISSUE TITLE
Spam score issue: "probable spam emails" still marked as spam inside the quarantine section with "Deliver"

### ISSUE THREAD

**Issue by luponata** [baruwaproject/baruwa2#138]
If you set baruwa to still deliver emails marked as "probable spam" (from 5 to 10 points, with default settings)

they are still marked as "probable spam" [pink color], and they are also inside the quarantine list, with the likelihood of confusion

**Comment by akissa** [comment]
Yes that is by design, messages will still get quarantined for records purposes.
